api/src/components/link/controller.ts

```diff
@@ -4,6 +4,14 @@ import * as publicationService from 'publication/service';
 
 import * as I from 'interface';
 
+export const get = async (event: I.APIRequest<undefined, undefined, I.GetLinksQueryParams>): Promise<I.JSONResponse> => {
+    
+    if(event.queryStringParameters) {
+        const links = await linkService.getPublicationLinks(event.queryStringParameters)
+        return response.json(200, links)
+    }
+    return response.json(404, 'nothing found')
```

Comment:
personally I prefer more context in an error and would include the missing publication id in the error (although some may say might lead to XSS issues I guess), hard to do here as you extract the publication id in the service

what happens if linkService throws an error?